code/modules/food_and_drinks/food/condiment.dm

```diff
@@ -16,6 +16,7 @@
 	//Possible_states has the reagent id as key and a list of, in order, the icon_state, the name and the desc as values. Used in the on_reagent_change(changetype) to change names, descs and sprites.
 	var/list/possible_states = list(
 	 "ketchup" = list("ketchup", "ketchup bottle", "You feel more American already."),
+	 "mustard" = list("mustard", "mustard bottle", "A spice mixed with enzymes and water."),
```

Comment:
I didn't know that the enzymes in ground mustard seeds turned the glucosinolates into isothiocyanate compounds with water. That's neat!